style.css

```diff
@@ -0,0 +1,47 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #d3d3d3;
+  height: 100vh;
+  width: 80%;
+  text-align: center;
+
+  margin-left: 10%;
+  margin-right: 10%;
```

Comment:
Use auto instead of %, auto will take the available space from the left and from the write.
suppose the screen is 800 px and the body Is 500.

`margin-left:auto`
`margin-right:auto`
like that, it will take the available space and divide it on the left and the right and the value will be : (800-500) / 2